nifi-api/src/main/java/org/apache/nifi/flow/VersionedProcessGroup.java

```diff
@@ -47,6 +47,9 @@ public class VersionedProcessGroup extends VersionedComponent {
     private Long defaultBackPressureObjectThreshold;
     private String defaultBackPressureDataSizeThreshold;
 
+    private Boolean logToOwnFile;
```

Comment:
I don't think we need this Boolean, do we? The presence or the absence of the log file suffix should be sufficient to determine this.

Reply:
Thanks @markap14 for your review.
For better User experience I've separated the enabling and configuration. There is a combolist with 'Yes' and 'No' options to turn dedicated logging and there is an input field with Process Group name as a default value for the log file name. I think this way the UI is more intuitive and more convenient than typing/deleting log file suffix.